frontend/src/pages/Recruits/Recruits.js

```diff
@@ -19,6 +20,7 @@ const BUTTON_LABEL = {
 
 const Recruits = () => {
   const { token } = useTokenContext();
+  const [courseStatus, setCourseStatus] = useState("전체");
```

Comment:
```suggestion
  const [courseTabStatus, setCourseTabStatus] = useState("전체");
```

r : 상수명(COURSE_TAB_LIST)라는 점에서, 네이밍을 조금 더 명시적으로 가져가는 것은 어떨까요?

Reply:
수정했습니다! 훨씬 의미적으로 명확하니 좋네요.